README.rst

```diff
@@ -109,24 +109,24 @@ where type must be one of ``[build, ci, docs, feat, fix, perf, refactor, revert,
 Note: only `fix`, `feat`, and `perf` will trigger a new NPM release, as this is the default behavior for semantic-release.
 
 Versioning and releases
-*****
+***********************
 
 This library has its version automatically updated by Lerna (i.e., ``lerna version``) using semantic-versioning under-the-hood when the release is published to npm. Lerna is configured to use independent versioning with conventional commits, as opposed to keeping all package versions in sync.
 
-When a PR is merged, you must manually run Lerna to create a release commit (e.g., ``chore(release): publish``). In this commit, Lerna increments the versions in the appropriate package.json files for any changed packages, creates Git tags, and updates the CHANGELOG file.
-To create this commit:
-::
-
-    npm run lerna:version
-    git push --set-upstream origin automation/lerna/version
-    git push —-tags
+When a PR is merged, you must manually run ``lerna version`` to create a release commit (e.g., ``chore(release): publish new versions``). In this commit, Lerna increments the versions in the appropriate package.json files for any changed packages, creates Git tags, and updates the CHANGELOG file(s).
 
-Once you have pushed the release commit and tag, you create a new PR from your branch ``automation/lerna/version`` in this example. Once the PR is open review and merge.
+Once your contribution's PR is approved/merged:
```

Comment:
I would switch the ordering around here to make it clearer that all of your steps below are to get a lerna-driven release commit, and that should happen after you merge your new code.